app/javascript/src/components/Dashboard/Contacts/Form/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+
+import { Formik, Form as FormikForm } from "formik";
+import { Check } from "neetoicons";
+import { Button, Pane, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+import { useTranslation } from "react-i18next";
+
+import { INITIAL_FORM_VALUES, ROLES, VALIDATION_SCHEMA } from "./constants";
+
+const Form = ({ onClose }) => {
+  const { t } = useTranslation();
+  const handleSubmit = async () => {
```

Comment:
Please add a newline between these. Eslint configs from Wheel should have taken care of this. Can you confirm whether your Eslint is using configs from Wheel?

Reply:
Yes, I am using the latest configs from Wheel.